### DISCUSSION TITLE
How would I create an "Animation Stopper" or "T Pose" script for Street Fighter 6?

### DISCUSSION THREAD

**Discussion by Libre-Ator** [praydog/REFramework#1162] [Scripting Questions]
I'm trying to extract my create a character using Ninja Ripper, but I can only get him in his current pose. Is there a way I can use REFramework to stop his animations from starting altogether, or force him to T pose or A pose or something?

**Comment by ItsGreggg** [comment]
Why on Gods earth has nobody responded to this. I’m trying to figure out the same. This makes zero sense. Were you able to find a way?

**Reply by Libre-Ator** [reply]
Haha, no, I'm sorry. I just extracted the CACs and made do with them. Though, I'm interested in learning how to make a script/mod like this if you can point me in the right direction. Also, I don't even own the game, but the demo assets gave me almost everything I needed for the CAC stuff. I just can't extract any morphed or edited characters, they just look like the standard body shapes.
